contributors/devel/api_changes.md

```diff
@@ -107,24 +109,34 @@ does not add a new required field)
      * which fields are required and which are not
      * mutable fields do not become immutable
      * valid values do not become invalid
+     * explicitly invalid values do not become valid
```

Comment:
Might be confusing for a novice - how does valid values in previous bullet contrast to explicitly invalid, etc

Reply:
s/explicitly/previously/ (if you meant something else, more explanation is needed)

Note that one might expect this and the prior line to have a symmetry over upgrade/downgrade, but they don't?

Reply:
What I wanted to convey was the difference between "we expanded the regex to include dashes" from the case of open-ended enum fields, which is discussed elsewhere in the doc.

Suggestions?